pkgs/applications/networking/cluster/zarf/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "zarf";
+  version = "0.26.1";
+
+  src = fetchFromGitHub {
+    owner = "defenseunicorns";
+    repo = "zarf";
+    rev = "v${version}";
+    hash = "sha256-ujidoLpqm7GaGeeTONFQ+SZNKG5Ei1W78DOqMZlRO7w=";
+  };
+
+  vendorHash = "sha256-vtfrcUKU4ELtJawJGugUxoQEgIZVjNNck8XzOyEWroo=";
+  preBuild = ''
+    mkdir -p build/ui
+    touch build/ui/index.html
+  '';
+
+  checkPhase = "";
```

Comment:
Is this an attempt to disable the `checkPhase`?

If you need to disable checks because they can't run in the sandbox or something, `doCheck = false` is the correct way to do that, but it would be nice to have tests if we can

Reply:
Thanks! and yes. I'll make that change.

Reply:
And apologies, I forgot to answer the second part of that. The tests make a bunch of network requests to pull down external resources, so I think we have to disable them.